webapp/src/main/webapp/vue-apps/sidebar/components/list/SidebarListFooter.vue

```diff
@@ -126,6 +126,12 @@ export default {
     productLink: eXo.env.portal.productLink,
     statusColor: '#707070'
   }),
+  created() {
+    document.addEventListener('user-status-updated', status => this.statusColor = status.color);
```

Comment:
Must be a function to be able to be removed in fact

```suggestion
    document.addEventListener('user-status-updated', this.changeStatusColor);
```

Reply:
@boubaker requested change is done , thanks

Reply:
Thanks, same for the second File

Reply:
Exact changes added as suggestion. In fact, the second parameter should be a function.
Besides, the consumed attribute will be of type [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event)

Thus, to access the object detail, you will need to access it via:
```vue
    changeStatusColor(event) {
      this.statusColor = event.detail.color;
    }
```

(PS: it's different from events via Vue using `this.$root.$emit` and `this.$root.$on`)

Reply:
Yes done too.

Reply:
Thanks